addons/mail/static/src/core/common/store_service.js

```diff
@@ -1,6 +1,7 @@
 import { Store as BaseStore, fields, makeStore, storeInsertFns } from "@mail/core/common/record";
 import { threadCompareRegistry } from "@mail/core/common/thread_compare";
 import { cleanTerm, generateEmojisOnHtml, prettifyMessageContent } from "@mail/utils/common/format";
+import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
```

Comment:
You cannot import a `web/` file in a `common/` file.
`common` is everywhere, while `web` is only available in backend.

One way to fix it is have code in function that you can patch in `web/store_service_patch.js`.